docs/src/pages/components/data-grid/columns/columns.md

```diff
@@ -34,43 +34,47 @@ By default, columns are ordered according to the order they are included in the
 > Otherwise, you take the risk of losing the column width state (if resized).
 > You can create the array outside of the render function or memoize it.
 
-## Column headers
+### Title & Description
```

Comment:
Title & description doesn't seem to macth the content

```suggestion
### Column headers
```

Reply:
Well I renamed it as this section is just about the title and description of the column headers.